### PR TITLE
Event cancellation (host)

### PR DESCRIPTION
**Pull request by ayhchen13**
- created new home page (index.html) displaying all verified events
- search function fully implemented (by artist, genre, venue, date)
- aesthetics improvement (dropdown from navbar,  images with corresponding event info)
- added host's event page with clickable button to cancel/edit events

### CODE REVIEW

File: PassIO/app.py [hunk 1]
```diff
@@ -32,7 +36,8 @@ def events_submit():
     genre = request.form.get('e_genre')
     tickets = mongo.db.Event.insert_one({'name': name, 'location': location, 'description': description, 
                                          'artist': artist, 'genre': genre, 
-                                         'verified': "false"})
+                                         'verified': "false", 'cancelled': False,
```
Comment (Jared-Waldroff) [inline PassIO/app.py:39]: Nice adding cancelled addition

File: PassIO/app.py [hunk 2]
```diff
@@ -143,23 +153,48 @@ def search():
...
         }
+        
+        # Mandate verified status
+        query = {
+                "$and": [
+                    query,  
+                    { "verified": "verified" }
+                ]
+        }
+
```
Comment (Jared-Waldroff) [inline PassIO/app.py:177]: Good implementation of searching for date.

File: PassIO/templates/navbar.html [hunk 3]
```diff
@@ -21,7 +22,22 @@
             <ul class="navbar-nav">
                 {% if CurrentUser %}
                 <!-- If user is logged in, show 'Account' -->
-                <li class="nav-item"> <a class="nav-link" href="/customerprofile">Account</a> </li>
+                <li class="nav-item dropdown">
```
Comment (Jared-Waldroff) [inline PassIO/templates/navbar.html:25]: Good addition of drop down here

**Review by Jared-Waldroff — APPROVED** [review]
Went through all code, addition of the drop down functions well. Changing of the home page and search functionality is good and works well. Addition of photos is good.

File: PassIO/app.py
Comment (BryceD-4) [inline PassIO/app.py:1]: Using index is good, we will just need to double check this is updated in our test files.

**Comment by BryceD-4** [comment]
I went through as well. Good to merge. We will just need to update our test cases with the changes in the .html pages (from events to index).

**Pull request merged**